### ISSUE TITLE
Support RecurrentGemma

### ISSUE THREAD

**Issue by julien-blanchon** [axolotl-ai-cloud/axolotl#1637]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/OpenAccess-AI-Collective/axolotl/discussions/categories/ideas) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/OpenAccess-AI-Collective/axolotl/labels/enhancement) didn't find any similar feature requests.

### 🔖 Feature description

Someone is working on adding support for RecurrentGemma ?

### ✔️ Solution

A jax finetune notebook from Google is available here: https://github.com/google-deepmind/recurrentgemma/blob/main/colabs/fine_tuning_tutorial_jax.ipynb

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this feature has not been requested yet.
- [X] I have provided enough information for the maintainers to understand and evaluate this request.

**Comment by winglian** [comment]
I don't know if we'll get to this soon. I would report upstream in HF transformers that recurrent Gemma isn't trainable

**Comment by NanoCode012** [comment]
It is supported upstream so Axolotl should work too https://github.com/huggingface/transformers/blob/main/src/transformers/models/recurrent_gemma/modeling_recurrent_gemma.py#L690
